.ci.yaml

```diff
@@ -2549,6 +2549,24 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac hello_world_macos__compile
+    bringup: true
+    recipe: devicelab/devicelab_drone
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "hostonly"]
+      task_name: hello_world_macos__compile
+
+  - name: Mac_arm64_ios hello_world_macos__compile
```

Comment:
@jmagman has been driving the effort to categorize different tests: host-only, devices, etc. to better use our bot resources. Let's land the infra change first so you can use directly: https://github.com/flutter/flutter/pull/110050

Reply:
Got it 👍

Reply:
@keyonghan I have changed this PRs `.ci.yaml` tests to use a `Mac_arm64` target. 
https://github.com/flutter/flutter/blob/d70df45883b810910e8b90a9e5aec1f750869b7b/.ci.yaml#L2561-L2568
This PR will be blocked until (https://github.com/flutter/flutter/pull/110050) lands, but we will not accumulate any technical debt.

Reply:
@a-wallen how about leaving off the arm test, and filing a new issue to add it once https://github.com/flutter/flutter/pull/110050  or similar lands?

Reply:
That's probably what is best now that https://github.com/flutter/flutter/pull/109889 seems to be blocked.

Reply:
@keyonghan I wrote a handful of draft PRs with `Mac_arm64` as a target. Do you have an idea of when `Mac_arm64` might be available?